github/tests/ReplayData/Repository.testEditWithAllArguments.txt

```diff
@@ -4,10 +4,10 @@ api.github.com
 None
 /repos/jacquev6/PyGithub
 {'Content-Type': 'application/json', 'Authorization': 'Basic login_and_password_removed', 'User-Agent': 'PyGithub/Python'}
-{"has_wiki": false, "name": "PyGithub", "has_downloads": true, "has_issues": true, "homepage": "http://vincent-jacques.net/PyGithub", "private": true, "description": "Description edited by PyGithub"}
+{"has_wiki": false, "name": "PyGithub", "has_downloads": true, "has_issues": true, "homepage": "http://vincent-jacques.net/PyGithub", "private": true, "description": "Description edited by PyGithub", "has_projects": false, "allow_squash_merge": true, "allow_merge_commit": true, "allow_rebase_merge": true}
```

Comment:
Manually editing the request/response pair is error prone and ugly, but it is necessitated by the fact that `self.repo` in the test points to a repository nobody else has the correct rights to, so whenever one needs to modify one of the existing tests there's no way to just re-record the exchange.